spring/boot3-actuator-autoconfigure/src/test/resources/application-autoConfTest.yml

```diff
@@ -17,3 +17,6 @@ management:
       base-path: /internal/actuator
       path-mapping:
         info: custom-info
+  endpoint:
+    heapdump:
+      access: unrestricted
```

Comment:
`heapdump` is restricted by default in Spring Boot 3.5
https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.5-Release-Notes#:~:text=Actuator%20'heapdump'%20Endpoint,leaking%20sensitive%20information.